content/docs/about/contributing.md

```diff
@@ -28,6 +28,7 @@ again.
 ### Follow the code of conduct
 
 Please make sure to read and observe our [Code of Conduct](https://github.com/kubeflow/community/blob/master/CODE_OF_CONDUCT.md).
+If you are a Kubeflow leader, please read our [inclusivity document](https://github.com/kubeflow/community/blob/master/INCLUSIVITY.md).
```

Comment:
I'm thinking the inclusivity guide is relevant to all community members. How about saying "... our Code of Conduct and inclusivity document."

Reply:
Thanks @sarahmaddox !
Sound good. I have modified the file. Please review/approve. Let me know if further change is needed.

Reply:
Thanks!